lib/travis/api/v3/billing_client.rb

```diff
@@ -102,6 +112,12 @@ def get_invoices_for_v2_subscription(id)
       end
     end
 
+    def get_invoices_for_v2_subscription(id)
```

Comment:
if it's billing related and not on master - don't merge

Reply:
Done